Plots/Tables/README.rst

```diff
@@ -0,0 +1,6 @@
+.. _table_examples:
+
+.. _table-examples-index:
+
+Tables
+============
```

Comment:
Please make the length of "=" signs exactly the same with above "Tables" string.